Readme.md

```diff
@@ -1 +1,59 @@
-This is for the code review of todo-app
+Note: This is for the code review of todo-app
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To-Do List:
+
+Additional description about the project and its features.
+-Phase 1. No features added yet.
+-Phase 2. Added CRUD features.
+-Phase 3. Added the functionality to tick mark completed tasks. Also added the functionality to remove all completed tasks.
+
+# Getting Started
+
+To get a local copy up and running follow these simple steps.
+
+- You can clone this repo by typing on your terminal.<pre><code>git clone https://github.com/muneebulrehman/ToDo-list.git</code></pre>
+- Type <pre><code>ToDo-list</code></pre> to access the project on the terminal
+
+## Built With
+
+- Major languages
+- HTML and CSS
+- Mostly JavaScript
+- WebPack used
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
```

Comment:
I think it will be better to fill out this section with instructions on how to get a copy of your project or removing it if it is unnecessary :memo:

Reply:
Fixed it. 😊